content/docs/components-and-props.md

```diff
@@ -156,11 +155,11 @@ function Comment(props) {
 
 [](codepen://components-and-props/extracting-components)
 
-It accepts `author` (an object), `text` (a string), and `date` (a date) as props, and describes a comment on a social media website.
+Acepta `author` (un objeto), `text` (una cadena), y `date` (una fecha) como propiedades, y representa un comentario en una web de redes sociales.
```

Comment:
We've talked about technical terms like `string` [here](https://github.com/reactjs/es.reactjs.org/pull/16). I would leave `string` as is.

Also, IMO "to _describe_" is not the same as "_to represent_". In React we describe components with JSX.
> ... y describe un comentario ...

instead of

> ... y representa un comentario ...